### PR TITLE
Support user output again

### LINKED ISSUE #108: No debug output in online editor anymore
Since #105 there is no user output (`echo` or `var_dump()`) forwarded to the online editor results. There is no test for that, so it wasn't catched in advance.

- [x] Add a test for user output
- [x] Make it possible again to have the output in the results

I also got that feedback from a collegue on CLI. Maybe PHPUnit needs to be configured accordingly?

### PR DESCRIPTION
**Pull request by mk-mxp**
I tried a lot of things to make PHPUnit render user output into JUnit logs. I then found out, that this is intentionally removed. So I looked for another way to get the output.

The output is contained in teamcity logs, with additional commandline settings. So I used this to add the output to the result.

Also added tests for the various conditions.

Closes #108

### CODE REVIEW

**Comment by homersimpsons** [comment]
For this I wanted to create a phpunit extension that would hook itself on phpunit and could do anything we want.

I feel like the current setup with junit export and parsing is quite "hacky".

File: junit-handler/src/TeamcityResult.php [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        $rawOutput = \mb_substr($line, $startOfOutput, $endOfOutput - $startOfOutput);
+
+        return $this->unescape($rawOutput);
+    }
+
+    private function unescape(string $text): string
+    {
+        return \str_replace(
+            [ "|'", '|n', '||', ],
+            [ "'",  "\n", '|', ],
```
Comment (homersimpsons) [inline junit-handler/src/TeamcityResult.php:91]: Should we assume that `|` is an escape char and just converting the `|(.)` to `\$1` would fit our needs?

Comment (mk-mxp) [reply junit-handler/src/TeamcityResult.php:91]: I haven't looked for a complete teamcity spec (for sure there is one). These seem to be the only escaped chars, so I used this collection. I also see this as a short term hacky workaround, until we agreed on doing some fundamental changes (which will take more than the 2h I spent making this work). I saw your #75 after writing #110, but it is the same idea.

Comment (mk-mxp) [reply junit-handler/src/TeamcityResult.php:91]: @homersimpsons `"\n"` must be the LF char, so `|(.)` -> `\$1` does not do the right thing.

Are you OK with keeping it like this?

**Comment by mk-mxp** [comment]
JetBrains [TeamCity message escaping](https://www.jetbrains.com/help/teamcity/service-messages.html#Escaped+Values) found. There is no Unicode [escaping done in PHPUnit](https://github.com/sebastianbergmann/phpunit/blob/main/src/Logging/TeamCity/TeamCityLogger.php#L331).

**Pull request merged**